Samples/Insights/cpp-win32/InsightsSample.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "WindowsAppSDKInsights.h"

```

Comment:
what does this provide? All you are using is from WIL.

Reply:
It provides the PartB information that will not be logged if you only used tracelogging.h from WIL.